Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM mirror.gcr.io/library/alpine:latest
 
+# Set up build arguments (provided by buildx)
+ARG TARGETOS
+ARG TARGETARCH
+
 # Install base tools and nginx
-RUN apk update && apk add --no-cache \
+  RUN apk update && apk add --no-cache \
```

Comment:
Can you remove this added space?  I don't think it causes any trouble, but gets non-standard